### DISCUSSION TITLE
@tailwind base; ^ syntax error: invalid or unexpected token.

### DISCUSSION THREAD

**Discussion by AvaniBaheti** [tailwindlabs/tailwindcss#13975] [Help] [answered]
I am getting this error when I am trying to run build for my NextJs and Tailwind CSS application.
I have latest versions of all the packages and I have created settings.json in .vscode folder. 
Can anybody suggest what can be done?

**Comment by Icegreeen** [comment, marked as answer]
Error: `**SyntaxError: invalid or unexpected token**`

In my case, I found the solution. It was for the following issue:

- **`I was importing Tailwind incorrectly into one of my components.`**

See the image below. I simply **`removed this import`** and the build error no longer occurred.

![Captura de tela de 2024-07-12 20-29-05](https://github.com/user-attachments/assets/f6074a66-7057-4d50-8d68-8c3c1c937114)

Solution: `Check your project to see if there is any improper import in your components or pages in the project.`

Version
```
"next": "^14.2.3",
"tailwindcss": "3.4.1",
```

**Reply by AvaniBaheti** [reply]
Worked... Thanks

**Reply by pasinduvinsuka** [reply]
Worked... Thank you ❤️

**Reply by Githarsh94** [reply]
Thankyou soo muchh
